apis/assets/site.webmanifest

```diff
@@ -1,14 +1,21 @@
 {
-    "name": "",
-    "short_name": "",
+    "name": "HiveGame",
```

Comment:
Not completely sure about the name of the Webapp. Should it be just "Hive" or "Hive Game" or something else?

Reply:
I'll keep that name, then. Thank you